### DISCUSSION TITLE
Can I share VPN connection with other hosts on same LAN?

### DISCUSSION THREAD

**Discussion by andreaippo** [qdm12/gluetun#877] [Q&A] [answered]
Hello,
I have currently gluetun running inside a docker container a new mini home server I bought and attached to my ISP's modem/router.
My house also has a Wireless router (working in AP mode) and other devices like PCs, smart TVs and smartphones.
My dream would be to be able to have all of those transparently connected to the gluetun VPN without any client side configuration, would that be even possible somehow?

Thanks!

**Comment by andreaippo** [comment, marked as answer]
> Hello, I have currently gluetun running inside a docker container a new mini home server I bought and attached to my ISP's modem/router. My house also has a Wireless router (working in AP mode) and other devices like PCs, smart TVs and smartphones. My dream would be to be able to have all of those transparently connected to the gluetun VPN without any client side configuration, would that be even possible somehow?
> 
> Thanks!

I guess that this answers my question:
#381

**Reply by AmyYummy** [reply]
** About VPN**
A Virtual Private Network (VPN) is an extension of a private network that includes a shared or public network link similar to the Internet. A VPN allows data to be sent between two computers over a shared or public network in a way that simulates a point-to-point private link.
If it is more popular, a VPN is actually a "line within a line", which is similar to a "bus line" on a city avenue. The difference is that the "line" composed of VPN does not exist physically, but It is simulated by technical means, that is, "virtual". However, this virtual private network technology can establish a logical dedicated "channel" for two computers in a public line, which has good confidentiality and non-disturbance, enabling both parties to conduct free and safe point-to-point connections, and are therefore very widely concerned by network administrators.
Through the virtual private network, we can easily and safely log in to the company's internal network at home, so that our dream of working from home can be realized.
In this article, the VPN server side uses Win2K; the client side uses Win98.
**Second, configure the VPN server**
1. Not yet configured: VPN in Win2K is included in "Routing and Remote Access Service". When your Win2K server is installed, it will automatically exist! But at this time, when you open the "Routing and Remote Access" item in "Administrative Tools" and enter its main window, select "Server Standard Status" in the "Tree" column on the left, and you can see its "Status" from the right is in a "stopped (not configured)" condition.
2. Start configuration: To allow Win2K computer to accept VPN dial-in from clients, the VPN server must be configured. In the left window, select "SERVER" (server name), right-click on it, and select "Configure and Enable Routing and Remote Access".
3. If this server has been configured before, and now you need to start over, right-click on "SERVER" (server name) and select "Disable Routing and Remote Access" to stop this service for reconfiguration!
4. After entering the configuration wizard, in "Public Settings", click to select "Virtual Private Network (VPN) Server", so that users can access this server through a public network (such as the Internet).
5. In the "Remote Client Protocol" dialog box, generally speaking, there should be at least the TCP/IP protocol, then just click "Yes, all available protocols are on the list" and then "Next" one step".
6. After that, the system will ask you to select another Internet connection used by this server, select the connection method used in the list under it (such as an established dial-up connection or a connection through a specified network card, etc.) and then "Next" ".
7. Then when answering the question "How do you want to assign IP addresses to remote clients", unless you have installed a DHCP server on the server side, please select "From a specified IP address range" here (recommended ).
8. Then enter the starting IP address you want to assign to the client according to the prompt, and "Add" it to the list, for example, "192.168.0.80~192.168.0.90" here. (Please note that this IP address range must be in the same network segment as the server's own IP address, that is, the previous "192.168.0" part must be the same!)
9. Finally, select "No, I don't want to set this server to use RADIUS" to complete the final setting. At this point, a small window for opening an account "Routing and Remote Access Service" will automatically appear on the screen. When it disappears, open "Services" in "Administrative Tools", and you can see "Routing and Remote Access" (Routing and Remote Access). Remote Access) item "Auto" is in "Started" state!
**Give the user the right to dial in**
1. By default, any user is refused to dial into the server.
2. To give a user the right to dial into this server, you need to open the user manager in the management tool (in the "Computer Management" item or "Active Directory Users and Computers"), select the required user, and in its Right-click on it and select "Properties".
3. Select the "Dial-in" item in the user's properties window, then click the "Allow access" item, and then "OK" to complete the work of granting the user's dial-in permission.

**VPN connection via LAN**
1. Enter the Win98 computer, if it wants to connect to the VPN server, it needs to install the "Virtual Private Network" service first. Under "Network" in the control panel, enter "Communication" to find this item and add it; after the installation is complete, restart the computer according to the prompts.
2. After restarting, there will be "Microsoft Virtual Private [Network](https;//www.baudcom.cn) Adapter" in the "Network" of the control panel, which means that the VPN service has been installed successfully!

Hope these point could help you.

Amy

**Reply by qdm12** [reply]
How is this relevant? Are you a bot??
